model/conversion_test.go

```diff
@@ -47,11 +47,15 @@ func TestProtoSchemaConversions(t *testing.T) {
 		"precedence": 5,
 		"route": [
 		{
-			"destination": "bar",
+			"destination": {
+                "name" : "bar"
```

Comment:
fix indent, here and below

Reply:
It looks fine in Gogland.. any idea why this is happening?

Reply:
Maybe tabs?

Reply:
Actually github messes up whitespace. It might be fine despite what web page shows.

Reply:
you have tabs you shouldn't use tabs